README.md

```diff
@@ -1,12 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio: mobile version
+# Portfolio: mobile version - Contact form
 
-> Portfolio: mobile version
+> Portfolio: mobile version - Contact form
 
 ![screenshot](./app_screenshot.png)
 
-This project is a mobile version for About-Me & Works sections of the portfolio.
+This project is a mobile version added contat form
 
 ## Built With
```

Comment:
Provide steps on how to install your project locally.

Reply:
It's already provided